data/repos.yml

```diff
@@ -117,6 +134,29 @@
   dashboard_url: https://grafana-paas.cloudapps.digital/d/xonj40imk/data-gov-uk?refresh=1m&orgId=1
   production_url: https://ckan.publishing.service.gov.uk/
 
+# ditto ^ https://github.com/alphagov/ckanext-datagovuk/blob/4ae4e0eb03c88b4526672527490a6546e95c4d78/docker/ckan/Dockerfile#L105-L112
+- repo_name: ckanext-dcat
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
+- repo_name: ckanext-harvest
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
+- repo_name: ckanext-spatial
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
```

Comment:
Same question here @kentsanggds

Reply:
I think we are still using this to address some issues with running CKAN on the old stack, the new stack is using the upstream version